deploy/examples/monitoring/rbac.yaml

```diff
@@ -7,12 +7,17 @@ metadata:
   name: rook-ceph-monitor
   namespace: rook-ceph
 rules:
-- apiGroups:
-  - monitoring.coreos.com
-  resources:
-  - '*'
-  verbs:
-  - '*'
+  - apiGroups:
+      - monitoring.coreos.com
+    resources:
+      - servicemonitors
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
`Watch` and `Delete` should not be needed.

Reply:
For read access we need `watch` also https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-example
`Delete` access of service monitor I am not sure

Reply:
We may need `patch`, but perhaps the `update` already covers updates during reconcile. I don't recall the difference.